lib/jit_preloader/version.rb

```diff
@@ -1,3 +1,3 @@
 module JitPreloader
-  VERSION = "0.3.0"
+  VERSION = "0.4.0"
```

Comment:
Should we take this opportunity to bump this to `1.0`, since it's been battle-tested and ready for production use for years now?

Reply:
sure